guides/common/modules/proc_removing_an_application_instance.adoc

```diff
@@ -0,0 +1,12 @@
+[id="{context}_removing_an_application_instance"]
+= Removing an Application Instance
+
+You can remove xref:{context}_application_instances[application instances] used for application centric deployment via your {ProjectServer} management UI.
+
+.Procedure
+. In the {ProjectWebUI}, navigate to *Application > App Instances*.
+. In the list of application instances, open the drop down menu and select *Delete* on the application instance you want to remove.
++
+This prompts a list of hosts that have been deployed via the selected application instance.
+You can choose to also delete hosts when deleting the application instance.
+Otherwise, those managed hosts remain.
```

Comment:
In general, we do not describe anything but the user actions, unless the user has to interact with, for example, the output. We trust that the user will see the list of hosts. I would remove this as it is out of sync with everything else.

Reply:
The user has to interact with it. There's a prompt to select individual hosts to be deleted. Else, hosts stay connected to Foreman.

No change.